news/2017-09-22-aspire-stage-three-osucatch-begins.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Aspire Stage Three: osu!catch Begins"
+date: 2017-09-22 00:00:00 +0100
+---
+With the osu! and osu!taiko communities flexing their creative muscles in the past two Aspire contests, the time is ripe for the fruit-catchers to take the spotlight, to the sound our very own Featured Artist, **OISHII**!
+
+![](/wiki/shared/news/banners/osu_catch.png)
+
+Casting down the Ranking Criteria and boldly venturing forth into a world where (almost) anything goes, Aspire is the premier "no-rules" official beatmapping contest.
+
+On two occasions already this year, we have seen some of the community's best and brightest mappers step up to this challenge. Between the somber tones of Helblinde's **The Solace of Oblivion** in osu! and the intensity of Function Phantom's **Algebra** in osu!taiko, Aspire has showcased some truly mind-blowing performances that have completely changed the landscape of modern mapping.
+
+Now it is time for the osu!catch community to join the fray, and as countless fruits rain down from above, the sky is no longer the limit! In an effort to give our extrodinarily talented Featured Artists the exposure they deserve, the osu!catch community will begin their contest with a track and artist that has excited many mappers since their unveiling:
+
+Courtesy of **OISHII**, take a trip down the *Onigiri Freeway*!
+
+<iframe width="700" height="394" src="https://www.youtube.com/watch?v=AKZd1zMxMlg?rel=0" frameborder="0" allowfullscreen></iframe>
+
+A fast and furious sugar-rush of synthesizers and sweetness, *Onigiri Freeway* presents a cheerful yet high-octane challenge for the  osu!catch community to tackle.
```

Comment:
thar be an extra space hiding before `osu!catch` here

Reply:
killed it